dev-docs/bidders/openweb.md

```diff
@@ -2,70 +2,89 @@
 layout: bidder
 title: OpenWeb
 description: Prebid OpenWeb Bidder Adapter
+pbjs: true
 biddercode: openweb
-media_types: banner
-tcfeu_supported: true
-userIds: britepoolId, criteo, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId
+media_types: banner, video
+multiformat_supported: will-bid-on-any
 schain_supported: true
 coppa_supported: true
+tcfeu_supported: true
+gpp_supported: true
+gpp_sids: tcfeu, usstate_all, usp
 usp_supported: true
-safeframes_ok: true
```

Comment:
If this is unsupported then set it to `false`, otherwise it will render as `check with bidder`

Reply:
Done

Reply:
@muuki88 any chance we can expedite the merge ?